serverless/containers/how-to/add-a-custom-domain-to-a-container.mdx

```diff
@@ -36,11 +36,69 @@ Each container comes with an automatically generated domain, on which it is avai
 
           - If your DNS provider does not support CNAME flattening, you need to create an ALIAS on your root domain (named `@` or with an empty name) instead of a CNAME.
     </Message>
-2. Click the **Endpoints** tab on your containers' page.
-3. Click **+ Add endpoint**.
-4. Enter your custom domain name and validate the form.
-5. The console runs checks on the CNAME and TLS configuration of the custom domain name. Connect to your container using your custom domain once all checks are complete.
+2. Before continuing, make sure the DNS entry is available. If not, the custom domain will likely end up in `error` on step 6. You can check this by:
+    - running `dig`, `drill` or `nslookup` commands on your domain using common DNS resolvers `1.1.1.1` and `8.8.8.8`: `dig <your_custom_domain> @1.1.1.1`, `dig <your_custom_domain> @8.8.8.8`
+    - or by using a DNS check tool of your choice (some are available online)
+3. Click the **Endpoints** tab on your containers' page.
+4. Click **+ Add endpoint**.
+5. Enter your custom domain name and validate the form.
+6. The console runs checks on the CNAME and TLS configuration of the custom domain name. See technical details below for more details.
```

Comment:
to me this is not a step

Reply:
Yes I agree, but since it was here before I've left it. Though this is misleading because it's not the console actually running the checks. I'll replace it with:

```suggestion
6. Wait while custom domain is being configured. See technical details below for more details.
```

Reply:
Well, more like in a note as @bene2k1 suggests below:

```suggestion
    <Message type="note">
      Before being `ready`, some checks are performed. See technical details below for more details.
    </Message>
```

Reply:
I've changed here: https://github.com/scaleway/docs-content/pull/3593/commits/428b80ba387f5ca600679096bdd5fd27a0d9de88

LMK what you think :+1: